### DISCUSSION TITLE
Too many requests error

### DISCUSSION THREAD

**Discussion by marksie1988** [BottlecapDave/HomeAssistant-OctopusEnergy#762] [Q&A]
I have just started getting this error, how do I reduce the number of requests to stop this happening? 

Maybe the integration needs some sort of rate limit so that it doesn't allow this to happen

```
Errors in request (https://api.octopus.energy/v1/graphql/): [{'message': 'Too many requests.', 'locations': [{'line': 2, 'column': 2}], 'path': ['smartMeterTelemetry'], 'extensions': {'errorType': 'AUTHORIZATION', 'errorCode': 'KT-CT-1199', 'errorDescription': 'There were too many requests. Please try again later.'}}]
```

**Comment by BottlecapDave** [comment]
This will occur if you have more than 2 meters, have increased the default frequency, are using the app excessively or have other integrations that are retrieving the data. You can adjust the frequency in your account config, as per the [docs](https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/setup/account/#refresh-rate-in-minutes)

I'll update the FAQ and home mini entities section with some more link backs.
